src/controllers/index.ts

```diff
@@ -96,6 +96,7 @@ export * as updateManageTrustsReviewLegalEntities from './update/update.manage.t
 export * as updateManageTrustsIndividualsOrEntitiesInvolved from './update/update.manage.trusts.individuals.or.entities.involved.controller';
 export * as updatePaymentFailed from './update/update.payment.failed.controller';
 export * as updateManageTrustsTellUsAboutTheLegalEntity from './update/update.manage.trusts.tell.us.about.the.legal.entity.controller';
+export * as statmentBOcease from './update/statement.bo.cease.controller';
```

Comment:
It is camelCase. BO is acronym and you do not write acronym's Bo because that doesn't represent what the word actually is. What is Bo? What is BO? BO is Beneficial Owner. So for better reading and understand point of you you don't write `statementBoCease` but `statementBOcease` as is stand for `statementBeneficialOwnersCease`